extensions/wikia/MercuryApi/MercuryApiHooks.class.php

```diff
@@ -85,9 +85,13 @@ public static function onArticleRollbackComplete( WikiPage $wikiPage, User $user
 	 * @return bool
 	 */
 	static public function onTitleGetSquidURLs( Title $title, Array &$urls ) {
+		global $wgServer;
+
 		if( $title->inNamespaces( NS_MAIN ) ) {
-			// Mercury API call from service
-			$urls[] = MercuryApiController::getUrl( 'getArticle', ['title' => $title->getPartialURL()] );
+			// Mercury API call from Ember to Hapi
+			$urls[] = $wgServer . self::SERVICE_API_BASE . self::SERVICE_API_ARTICLE . $title->getPartialURL();
```

Comment:
`wfExpandUrl` would only add `$wgServer`. As @bognix wrote - please add an example URL in the comment above.

Making a method and unit tests for it will be beneficial when migration to the service takes place. I understand that it's a temporary solution, but let's not introduce technical dept when it can be easily avoided.